app/react/Layout/DocumentsList.js

```diff
@@ -77,25 +84,26 @@ class DocumentsList extends Component {
       connectionsGroups,
       LoadMoreButton,
       rowListZoomLevel,
+      CollectionViewer,
     } = this.props;
-    let counter = (
-      <span>
-        <b>{documents.get('totalRows')}</b> <Translate>documents</Translate>
-      </span>
+
+    const totalConnections = connections
```

Comment:
@mfacar should we include this piece of code into the DocumentCounter component?

Reply:
> @mfacar should we include this piece of code into the DocumentCounter component?
Yes, we could do it, in that case we have to pass the connections and connectionsGroups objects to the DocumentCounter, I didn't include before because I tried to pass only numeric values, but since there isn't any other use of totalConnections in DocumentList it makes total sense to move into the specific component

Reply:
@mfacar well, you're right. It makes sense to make the calculation on the DocumentList also, since the DocumentCounter is merely presentational.